digits/extensions/view/__init__.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+from . import rawData
+
+view_extensions = [
+    # set show=True if extension should be listed in known extensions
+    {'class': rawData.Visualization, 'show': True},
```

Comment:
Seems like we should also show the gradient datasets if we're showing the models